Non_Uniform_Sampling/Bruker_NUS/4D.md

```diff
@@ -73,24 +91,14 @@ Tested environment:
 
     ![screenshot](../images/navigation_4D.png)
 
-
-13. If the phasing or other processing parameters are off, adjust them and repeat the reconstruction. If the projections looks good - congratulations!
+13. If the phasing or other processing parameters are off, adjust them and repeat the NUS reconstruction. If the 
+projections look good - congratulations!
 
 ## Troubleshooting 
 
 ### Post-reconstruction phasing
-If the spectrum appears to be misphased, it has to be processed all over again. 
-To determine the optimal phasing angles:
-* Navigate to your peak list. Note down the positions of several peaks you want to phase (in points). 
-  * If the specrum lines are in dispersion shape, i.e. misphased by 90°, you may have to fine tune the peak picking parameters to get the positions of the most strongest peaks.  
-* `rpl` - to Read PLane from the 4D spectrum. Set the plane numbers to the positions of your selected peaks. 
-  * As such, the command `rpl 34 75 70 347570` will extract a 34 (which is, usually, HN) plane at the point 75 in F1 and 70 in F2 and write it into the procedure 347570 
-*  `xif1` and `xif2` to execute the inverse FT, thereby restoring the imaginary part necessary for phasing
-*  `xfb` to execute FT and get back to the frequency domain
-*  `.ph` to enter the interactive phasing mode
-*  Note down the values of PH0 and PH1. Modify the `PHC0`, `PHC1` and `MddPHASE` paramenters accordingly. 
-* Repeat the same in all dimensions (e.g. 21 planes) and do not rely on phasing the single peak: make sure to test several planes, and perhaps not only the planes with peak maxima but also peak borders!
-* Reinitiate the NUS reconstruction. 
+If the spectrum appears to be misphased, it has to be processed all over again. To determine the optimal phasing angles:
+
```

Comment:
Missing reference link to the 4D phase correction tutorial. 

Add something like:

"see [the 4D phase correction tutorial](../../TOPSPIN/Phase_Correction/4D.md)"

(pls double-check the link)

Reply:
I will add the link in the next merge request because this file does not exist in that location anymore.